code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -823,12 +823,19 @@ All effects don't start immediately, but rather get worse over time; the rate is
 /datum/reagent/consumable/ethanol/manhattan_proj/on_mob_life(mob/living/carbon/M)
 	M.set_drugginess(30)
 	if(isethereal(M))
+		to_chat(M, span_notice("Danger! Danger! High Voltage!! When we drink..."))
 		var/mob/living/carbon/C = M
 		var/obj/item/organ/stomach/ethereal/stomach = C.getorganslot(ORGAN_SLOT_STOMACH)
 		if(istype(stomach))
 			stomach.adjust_charge(M.reagents.get_reagent_amount(/datum/reagent/consumable/ethanol/manhattan_proj) * REM * ETHEREAL_CHARGE_SCALING_MULTIPLIER)
 	return ..()
 
+/datum/reagent/consumable/ethanol/manhattan_proj/reaction_mob(mob/living/M, method=INGEST)
+	if(method != INGEST)
```

Comment:
```suggestion
/datum/reagent/consumable/ethanol/manhattan_proj/reaction_mob(mob/living/M, method=TOUCH)
	if(method == INGEST)
```

no reason to use method=INGEST rather than method=TOUCH and it just adds inconsistency
also, you want it to happen specifically when the method IS ingest, not when it isn't
make this change for all the others

Reply:
All done and updated to be the correct way. Thank you.